mobify/platform-scaffold
# Add-2-Home-Screen Headerbar Improvements

# Add-2-Home-Screen Headerbar Improvements

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1497
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/826

## Changes
- Added `Back` button in A2HS headerbar
- Styled the headerbar differently on A2HS Mode

## Todos

## How to test-drive this PR
- `npm link` to the SDK `a2hs-headerbar` branch (https://github.com/mobify/progressive-web-sdk/pull/826)
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Navigate around, ensuring Back button works
- Ensure styles match the mocks
